source/refined-github.css

```diff
@@ -154,3 +154,13 @@ pr-branches
 	margin-left: 4px !important;
 	margin-right: 4px !important;
 }
+
+/* Style backticked code in links #4817 */
+.js-commits-list-item code .markdown-title,
+code .markdown-title[data-test-selector='commit-tease-commit-message'] {
```

Comment:
Let's comment where these are found, in a style similar to `parse-backticks`’ comments

Reply:
Also, why style the links? The CSS can be on `code`, copy-pasted from GitHub’s own working selector 

```css
padding: 2px 4px;
background-color: var(--color-neutral-muted);
font-size: .9em;
line-height: 1;
border-radius: 6px;
```

<img width="536" alt="Screen Shot 4" src="https://user-images.githubusercontent.com/1402241/134769463-ff3d6f8d-ac3a-4c1f-9ade-8b1ce262f806.png">

Here it is, applied to `code` (upper) and the native version (lower) 
<img width="587" alt="Screen Shot 3" src="https://user-images.githubusercontent.com/1402241/134769469-113c200c-b5a7-48f8-a611-c9b64ab28690.png">

Reply:
> Also, why style the links?

Because `code` is ambiguous. Or do you mean we just style all `code`, regardless whether they are already styled by GitHub or not?

Reply:
You are right, we should style `code` instead, because I see this:

https://github.com/sindresorhus/refined-github/search?q=linkify&type=commits